hwy/ops/rvv-inl.h

```diff
@@ -3141,6 +3141,11 @@ HWY_API VFromD<D> SumOfLanes(D d, const VFromD<D> v) {
   return detail::RedSum(d, v, v0);
 }
 
+template <class D>
+HWY_API TFromD<D> ReduceSum(D d, const VFromD<D> v) {
```

Comment:
On RVV we can avoid this GetLane but no worries, I will take care of it after we merge.